docs/operation/upgrade.rst

```diff
@@ -0,0 +1,94 @@
+Upgrade Guide
+=============
+This document describes a complete and reliable upgrade path for **MetalK8s**
+and all the components that make up the stack.
+
+.. important::
+
+    - Upgrading MetalK8s is a manual process and as a result,
+      the following outlined steps must be met to the later in other to achieve
+      a successful upgrade.
+    - The output of the below-provided commands solely depends on the particular
+      MetalK8s setup you have in place.
+
+.. warning::
+    For any cluster upgrade from MetalK8s v2.0 to v2.1 and on-ward, **Kubelet**
+    must be upgraded before making any upgrades to the salt-master. Once this
+    is done, an upgrade is guaranteed to proceed in a seamless manner.
+
+Supported Versions
+******************
+
+This guide currently supports MetalK8s upgrades for the following versions.
+Please refer to the release notes for more information.
+
+    - Upgrade 2.0.0 to 2.1.0
+    - Upgrade 2.1.0 to 2.2.0
+
+
+Upgrade Steps
+*************
+This section describes the logical steps for Upgrading MetalK8s.
+
+#. Connect to the :term:`Bootstrap node`.
+
+#. Download the new `.ISO` from `Sources <http://scality.com>`_.
```

Comment:
I don't think we will give a link to download ISOs as it's open source and they all can build it themelse. We should just say `Upload(or Get or Whatever)  the new ISO to the Bootstrap node` or something like that

Reply:
I think it will be much easier if we could generate releases for the various Metalk8s versions and add then add them to our github releases. IMPOV
 
Not the least, I will make the changes you have suggested

Reply:
Yes I agree but I think it is not planned for the moment